_includes/layouts/info-page.html

```diff
@@ -6,17 +6,115 @@
 This template is for longer information pages
 {% endcomment %}
 
+
+{% comment %}
```

Comment:
Blocking: If we have to duplicate this code, you should put comments on both sides in all caps so that we do not update this in one place and miss it in the other.  I'm particularly thinking about that comment on Ticket #267.

But first, see next comment on trying things to avoid this particular code duplication.